src/main/groovy/org/edx/jenkins/dsl/UserRetirementConstants.groovy

```diff
@@ -0,0 +1,115 @@
+    }
+
+    public static def common_triggers = { extraVars ->
+        if (extraVars.containsKey('CRON')) {
+            cron(extraVars.get('CRON'))
+        }
+    }
+
+    public static def common_wrappers = { extraVars ->
+        return {
+            buildUserVars() /* gives us access to BUILD_USER_ID, among other things */
+            buildName('#${BUILD_NUMBER}, ' + extraVars.get('ENVIRONMENT_DEPLOYMENT'))
+            environmentVariables {
+                env('ENVIRONMENT_DEPLOYMENT', extraVars.get('ENVIRONMENT_DEPLOYMENT'))
+            }
+            timestamps()
+            colorizeOutput('xterm')
+        }
+    }
+
+    public static def common_parameters = { extraVars ->
+        return {
+            stringParam('TUBULAR_BRANCH', 'master', 'Repo branch for the tubular scripts.')
+            stringParam('USER_RETIREMENT_SECURE_BRANCH', 'master', 'Repo branch for the tubular scripts.')
+        }
+    }
+
+    public static def common_multiscm = { extraVars ->
+        return {
+            git {
+                remote {
+                    url('git@github.com:edx-ops/user-retirement-secure.git')
+                    if (extraVars.containsKey('SECURE_GIT_CREDENTIALS')) {
+                        credentials(extraVars.get('SECURE_GIT_CREDENTIALS'))
+                    }
+                }
+                branch('$USER_RETIREMENT_SECURE_BRANCH')
+                extensions {
+                    relativeTargetDirectory('user-retirement-secure')
+                    cloneOptions {
+                        shallow()
+                        timeout(10)
+                    }
+                    cleanBeforeCheckout()
+                }
+            }
+            git {
+                remote {
+                    url('https://github.com/edx/tubular.git')
+                }
+                branch('$TUBULAR_BRANCH')
+                extensions {
+                    relativeTargetDirectory('tubular')
+                    cloneOptions {
+                        shallow()
+                        timeout(10)
+                    }
+                    cleanBeforeCheckout()
+                }
+            }
+        }
+    }
+    public static def common_publishers = { extraVars, job_name ->
+        return {
+            // After all the build steps have completed, cleanup the workspace in
+            // case this worker instance is re-used for a different job.
+            wsCleanup()
+
+            if (extraVars.containsKey('MAILING_LIST')) {
+                // Send an alerting email upon failure.
+                extendedEmail {
+                    recipientList(extraVars.get('MAILING_LIST'))
+                    triggers {
+                        failure {
+                            attachBuildLog(false)  // build log contains PII!
+                            compressBuildLog(false)  // build log contains PII!
+                            subject('Failed build: ' + job_name + ' #${BUILD_NUMBER}')
```

Comment:
nit: We can probably use JOB_NAME environment variable like you use BUILD_NUMBER and skip adding the method parameter altogether.